gordo/machine/model/factories/feedforward_autoencoder.py

```diff
@@ -88,8 +88,9 @@ class (e.x. Adam(lr=0.01,beta_1=0.9, beta_2=0.999)).  If no arguments are
 
     # Instantiate optimizer with kwargs
     if isinstance(optimizer, str):
-        Optim = getattr(keras.optimizers, optimizer)
-        optimizer = Optim(**optimizer_kwargs)
```

Comment:
I think this previous approach still works though. It's what we have implemented in aion and runs fine. But I'm ok with changing it.